course_content/iris_course/5.Cube_Plotting.ipynb

```diff
@@ -384,10 +383,17 @@
     "## 5.4 Section Summary: Cube Plotting<a id='summary'></a>\n",
     "\n",
     "In this section we learnt:\n",
-    "* `iris.plot` and `iris.qplt` are used to create plots of the data stored in iris Cubes.\n",
-    "* `iris.qplt` automatically adds a plot title, axis titles, and a colour bar when appropriate\n",
+    "* `iris.plot` and `iris.quickplot` are used to create plots of the data stored in iris Cubes.\n",
+    "* `iris.quickplot` automatically adds a plot title, axis titles, and a colour bar when appropriate\n",
     "* Cartopy is used to project the Cube's data on to different map projections."
    ]
+  },
+  {
```

Comment:
An extra blank cell has crept back in.
This keeps happening, perhaps we should just leave them ?

Reply:
Yeah I don't think it really matters if we leave extra cells at the end. But we may as well remove them if we find them.